.gitignore

```diff
@@ -1,3 +1,7 @@
 /target/
 **/*.rs.bk
-Cargo.lock
+Cargo.loc
+
+# vim cached rescue files
+**.swp
+**.swo
```

Comment:
Can you please revert these changes?
You can configure Vim to keep swap files in a separate directory if you add something like this to your `vimrc`:

```
set dir=$HOME/.nvim/tmp
```

Reply:
thanks, I will do.